Specs/Core/SerializedMapProjectionSpec.js

```diff
@@ -50,9 +54,12 @@ defineSuite([
     });
 
     it('serializes and de-serializes Proj4Projection', function() {
-        var projection = new Proj4Projection('+proj=moll +lon_0=0 +x_0=0 +y_0=0 +a=6371000 +b=6371000 +units=m +no_defs', 0.5);
+        var proj4Uri = '../../../ThirdParty/proj4js-2.5.0/proj4.js';
+        var projection = new Proj4Projection(proj4Uri, proj4, '+proj=moll +lon_0=0 +x_0=0 +y_0=0 +a=6371000 +b=6371000 +units=m +no_defs', 0.5);
         var serialized = new SerializedMapProjection(projection);
 
+        spyOn(SerializedMapProjection, '_getProj4').and.returnValue(when.resolve(proj4));
```

Comment:
The URI above is actually valid, and removing this spy lets tests pass in un-built Cesium, but in `release` built Cesium this test fails without the spy.